### PR TITLE
Add webhook events and payloads to menu

### LINKED ISSUE #216: Write documentation for Webhook events and payloads component
We need to write documentation on the Webhook events and payloads component.

 https://devdocs.mautic.org/en/5.x/webhooks/events/index.html

### PR DESCRIPTION
**Pull request by ifeoluwafavour**
Resolves #216 - documentation for webhook events and payloads

### CODE REVIEW

File: docs/webhooks/events/index.rst
Comment (RCheesley) [inline docs/webhooks/events/index.rst:1]: @ifeoluwafavour when I build the HTML there's a bunch of warnings that these pages do not exist in a ToC tree.

> /Users/rcheesley/Sites/local.newdevdocs/developer-documentation-new/docs/webhooks/events/company_post_delete.rst: WARNING: document isn't included in any toctree
> /Users/rcheesley/Sites/local.newdevdocs/developer-documentation-new/docs/webhooks/events/company_post_save.rst: WARNING: document isn't included in any toctree
> /Users/rcheesley/Sites/local.newdevdocs/developer-documentation-new/docs/webhooks/events/email_on_open.rst: WARNING: document isn't included in any toctree
> /Users/rcheesley/Sites/local.newdevdocs/developer-documentation-new/docs/webhooks/events/email_on_send.rst: WARNING: document isn't included in any toctree
> /Users/rcheesley/Sites/local.newdevdocs/developer-documentation-new/docs/webhooks/events/form_on_submit.rst: WARNING: document isn't included in any toctree
> /Users/rcheesley/Sites/local.newdevdocs/developer-documentation-new/docs/webhooks/events/lead_channel_subscription_changed.rst: WARNING: document isn't included in any toctree
> /Users/rcheesley/Sites/local.newdevdocs/developer-documentation-new/docs/webhooks/events/lead_company_change.rst: WARNING: document isn't included in any toctree
> /Users/rcheesley/Sites/local.newdevdocs/developer-documentation-new/docs/webhooks/events/lead_points_change.rst: WARNING: document isn't included in any toctree
> /Users/rcheesley/Sites/local.newdevdocs/developer-documentation-new/docs/webhooks/events/lead_post_delete.rst: WARNING: document isn't included in any toctree
> /Users/rcheesley/Sites/local.newdevdocs/developer-documentation-new/docs/webhooks/events/lead_post_save_new.rst: WARNING: document isn't included in any toctree
> /Users/rcheesley/Sites/local.newdevdocs/developer-documentation-new/docs/webhooks/events/lead_post_save_update.rst: WARNING: document isn't included in any toctree
> /Users/rcheesley/Sites/local.newdevdocs/developer-documentation-new/docs/webhooks/events/page_on_hit.rst: WARNING: document isn't included in any toctree
> /Users/rcheesley/Sites/local.newdevdocs/developer-documentation-new/docs/webhooks/events/sms_on_send.rst: WARNING: document isn't included in any toctree
> done

Can you please add them, maybe to the root index.rst under the Webhooks section? That way they also come up under the menu when you're clicking through. Thanks!

**Review by RCheesley — CHANGES_REQUESTED** [review]
Some build errors as the webhook pages aren't on any ToC tree. Please take a look!

File: docs/webhooks/events/index.rst
Comment (ifeoluwafavour) [reply docs/webhooks/events/index.rst:1]: @RCheesley i have just added the toctree. Is the syntax okay?

**Review by RCheesley — APPROVED** [review]
This looks great @ifeoluwafavour sorry for the delay in getting back to you!

**Pull request merged**